addons/website_event/views/event_templates_page.xml

```diff
@@ -9,65 +9,103 @@
-                    <span class="navbar-toggler-icon"></span>
-                </button>
-                <div id="o_wevent_event_submenu" class="collapse navbar-collapse">
-                    <ul class="navbar-nav flex-md-wrap w-100">
-                        <t t-foreach="event.menu_id.child_id" t-as="submenu">
-                            <t t-call="website.submenu">
-                                <t t-set="item_class" t-value="'nav-item'"/>
-                                <t t-set="link_class" t-value="'nav-link'"/>
+
+        <!-- Desktop -->
+        <div id="o_wevent_submenu_desktop" class="d-none d-lg-block mt-3 mb-3 mb-lg-2">
+            <div class="container">
+                <div class="d-flex align-items-baseline justify-content-between">
+                    <nav class="d-flex flex-wrap justify-content-between align-items-center gap-2 flex-grow-1">
+                        <ul class="breadcrumb p-0 m-0">
+                            <li class="breadcrumb-item">
+                                <a href="/event" title="Back to all events">All Events</a>
+                            </li>
+                            <t t-if="navbar__back_url">
+                                <li class="breadcrumb-item">
+                                    <a t-attf-href="/event/{{slug(event)}}" title="Back to all events" t-field="event.name"/>
+                                </li>
+                                <li class="breadcrumb-item">
+                                    <a t-att-href="navbar__back_url" t-att-title="navbar__back_title" t-out="navbar__back_text"/>
+                                </li>
+                            </t>
+                            <t t-else="">
+                                <li class="breadcrumb-item active" aria-current="page"><span class="pe-3" t-field="event.name"/></li>
                             </t>
-                        </t>
-                    </ul>
+                        </ul>
+                        <ul t-if="not navbar__back_url" class="nav" t-att-data-menu_name="editable and 'Event Menu'" t-att-data-content_menu_id="editable and event.menu_id.id">
+                            <t t-foreach="event.menu_id.child_id" t-as="submenu">
+                                <t t-if="(submenu == event.register_menu_ids.menu_id) and (not is_rendering_cta and not hide_register_cta)" t-call="website.submenu">
+                                    <t t-set="item_class" t-value="'nav-item'"/>
+                                    <t t-set="link_class" t-value="'nav-link text-nowrap'"/>
+                                </t>
+                                <t t-elif="(submenu != event.register_menu_ids.menu_id)" t-call="website.submenu">
+                                    <t t-set="item_class" t-value="'nav-item'"/>
+                                    <t t-set="link_class" t-value="'nav-link text-nowrap'"/>
+                                </t>
```

Comment:
`submenu != event.register_menu_ids.menu_id or not is_rendering_cta` 

Not really because, with OR,   the condition could be `true` even in Register pages... but you gave me an idea to streamline the condition and remove the need for the double t-call, thanks!

Reply:
Can keep the current condition as it's very clear but for :nerd_face: purposes I still feel the need to argue the other condition should have worked fine
```
For R "The menu is the register menu", CTA "the CTA is rendering", the condition "whether the menu should be rendered"
R CTA     !R | CTA
0  0          !0 | 0 = 1 | 0 = 1
0  1          !0 | 1 = 1 | 1 = 1
1  0          !1 | 0 = 0 | 0 = 0   don't render if it's the register menu and we're not rendering the CTA
1  1          !1 | 1 = 0 | 1 = 1   render in all other cases
```

Gave the code another look this morning, still nothing of note to change for me, if you want to send it to Luc for testing (if not done yet)